tutorials/01-gaussian-mixture-model/index.qmd

```diff
@@ -109,14 +106,20 @@ We run a MCMC simulation to obtain an approximation of the posterior distributio
 We use a `Gibbs` sampler that combines a [particle Gibbs](https://www.stats.ox.ac.uk/%7Edoucet/andrieu_doucet_holenstein_PMCMC.pdf) sampler for the discrete parameters (assignments $k$) and a Hamiltonion Monte Carlo sampler for the continuous parameters ($\mu$ and $w$).
 We generate multiple chains in parallel using multi-threading.
 
-```julia
+```{julia}
+#| output: false
+setprogress!(false)
+```
+
+```{julia}
 sampler = Gibbs(PG(100, :k), HMC(0.05, 10, :μ, :w))
 nsamples = 100
 nchains = 3
 chains = sample(model, sampler, MCMCThreads(), nsamples, nchains);
 ```
 
```

Comment:
```suggestion
::: {.callout-warning collapse="true"}
## Sampling With Multiple Threads

The `sample()` call above assumes that you have at least `nchains` threads available in your Julia instance. If you do not, the multiple chains 
will run sequentially, and you may notice a warning. For more information, see [the Turing documentation on sampling multiple chains.](https://turinglang.org/dev/docs/using-turing/guide/#sampling-multiple-chains)
:::
```

Explain the warning they may get if they run this notebook with one thread. This is pretty generic and we can do this for all the similar warnings.